snap7/types.py

```diff
@@ -246,6 +246,19 @@ def __str__(self) -> str:
 
 
 class S7CpuInfo(ctypes.Structure):
+    """
```

Comment:
According to [PEP](https://peps.python.org/pep-0257/#multi-line-docstrings), a docstring should follow this format: summary, followed by a blank line, and then the rest of the content.

Reply:
Ok, I will made it after easter,
Happy Holidays